doc/contributing/PROCESS.md

```diff
@@ -174,6 +174,96 @@ To make a new OpenAssetIO release, follow this procedure.
+these as both design validators and testing coverage providers.
+
+In order to tie this coverage into mainline OpenAssetIO development,
+OpenAssetIO continuous integration checks out several downstream
+projects each time a commit is pushed. These projects are built against
+the bleeding edge version of OpenAssetIO, and their test suites are run.
+This provides valuable end to end testing of each changeset, and is
+invaluable. (See
+[integrations.yml](https://github.com/OpenAssetIO/OpenAssetIO/blob/main/.github/workflows/integrations.yml))
+
+Furthermore, as these downstream repos are projects in their own right,
+they also have their own CI suites to validate any changes made to them,
+using a released OpenAssetIO version, taken from PyPI.
+
+### Breaking change mitigation process
+
+On occasion, it is necessary for OpenAssetIO to change its interfaces in
+such a way that cause the downstream projects to become incompatible
+against the bleeding edge OpenAssetIO. You will know when this happens
+as one of the tests in
+[integrations.yml](../../.github/workflows/integrations.yml) will begin
+to fail.
+
+The downstream projects are pinned to a specific version of OpenAssetIO
+on PyPI for their own test suites, so these will continue to function
+even through a breaking change on OpenAssetIO main. However, the
+OpenAssetIO integrations checks will being to fail, with no clear way to
+remedy the situation before an OpenAssetIO release, as any changes to
+the downstream project to support bleeding edge will likely cause
+incompatibility with the published OpenAssetIO release the project is
+consuming from PyPI.
+
+In order to remedy this, when making a breaking OpenAssetIO change that
+causes a failing integrations check, the following procedure may be
+implemented to avoid long term loss of coverage.
+
+1. Consult on whether the change must necessarily be breaking, if
+   reasonably possible, attempt to formulate another approach.
+
+2. Evaluate the scope of breakage in the dependent project(s), and make
+   an issue in those projects to fix compatibility with your change.
+
+3. Comment out the failing integrations check in
+   [integrations.yml](../../.github/workflows/integrations.yml). Leave a
+   note explaining the context, with links to the issues created in the
+   previous step.
+
+4. Review and merge your change to OpenAssetIO as normal, call out the
+   removed integrations check in the PR.
+
+5. As soon as possible, plan to work on the issues(s) created in step 2
+   to return compatibility to the dependent repos.
+
+6. To accomplish this, branch the dependent repo in question from main,
+   call your new branch `openassetio/X.Y.Z`, where `X.Y.Z` is the *next*
+   release of OpenAssetIO.
+
+   > **Warning**
+   >
+   > The specific branch name formatting is important, as CI is setup to
+   > source wheels built from the tip of `openassetio:main` when run on
+   > these  branches
```

Comment:
[consider] I'm not sure quite where to put this, but it feels like we should highlight that a downstream project _may_ not have CI already configured to pull `main` wheels for dep feature branches as described.

I imagine that the first time we need to go through this process, we'll add the CI tweaks, vs up front.

Reply:
Yeah, I was writing this document from the perspective of the future, in the hopes that it would become true in time and we wouldn't have to muddy the document, as it's already quite complex.

However, as it's possible we won't get to doing this on any specific repo for a while, I'll add a clarifying sentence.

Reply:
[3fb73ab](https://github.com/OpenAssetIO/OpenAssetIO/pull/772/commits/3fb73ab95cc522add5fb7e1159467e8d3d797758)